103_lineshape_broadening.ipynb

```diff
@@ -0,0 +1,874 @@
+    "# let's do the same at higher temperature : \n",
+    "\n",
+    "hwhm_doppler_300 = doppler_broadening_HWHM(df.wav, molar_mass=29, Tgas=300)\n",
+    "hwhm_doppler_1000 = doppler_broadening_HWHM(df.wav, molar_mass=29, Tgas=100)\n",
+    "hwhm_doppler_4000 = doppler_broadening_HWHM(df.wav, molar_mass=29, Tgas=4000)\n",
+    "\n",
+    "I_300 = gaussian_lineshape(w - w_center, hwhm_doppler_300)\n",
+    "I_1000 = gaussian_lineshape(w - w_center, hwhm_doppler_1000)\n",
+    "I_4000 = gaussian_lineshape(w - w_center, hwhm_doppler_4000)\n",
+    "\n",
+    "# Let's plot \n",
+    "plt.figure()\n",
+    "plt.plot(w, I_300, label=\"300 K\")\n",
+    "plt.plot(w, I_1000, label=\"1000 K\")\n",
+    "plt.plot(w, I_4000, label=\"4000 K\")\n",
+    "plt.xlabel(\"Wavenumber (cm-1\")\n",
+    "plt.ylabel(\"Doppler Lineshape\")\n",
+    "plt.legend()\n",
+    "plt.xlim((2298.4, 2298.5))\n",
+    "# %%"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0dd2e77b",
+   "metadata": {},
+   "source": [
+    "The broadening increases with the temperature , however it remains very low (< 0.02 cm-1) even at extremelly high temperatures of 4000 K."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0da7e95e",
+   "metadata": {},
+   "source": [
+    "# Collisional Broadening\n",
+    "If a molecule is disturbed by a collision, this shortens the lifetime of the level, ie the decay rate is increased. This will increase the uncertainty in the emitted/absorbed energy, so that photons at a frequency $ ν_0 $ ± Δν can also be absorbed or emitted. This broadening factor is linked to the number of collisions, and therefore it is linked to the pressure,the temperature, and the nature of the colliding partner. \n",
+    "\n",
+    "It is also called pressure broadening. Pressure broadening mainly takes place in dense environments like stellar and\n",
+    "planetary atmospheres and dominates over Doppler Broadening.Collisions between molecules or atoms, induce pressure\n",
+    "which can supply or remove small amounts of energy during radiative transitions, thereby allowing photons \n",
+    "with a broader range of frequencies to produce a particular transition of a molecule.\n",
+    "\n",
+    "Collision broadening is most significant at High Pressure, low Temperature, large Wavelength."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ae107f2",
+   "metadata": {},
+   "source": [
+    "The corresponding line has a Lorentzian profile, "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17c1b056",
+   "metadata": {},
+   "source": [
+    "# $                        ϕ(ν)= \\frac{1}{\\pi} \\frac{\\gamma_{lb}}{\\gamma_{lb}^2+w_{centered}^2}$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57ccf3af",
+   "metadata": {},
+   "source": [
+    "At the first order it is proportional to the pressure and to a coefficient :\n",
+    "\n",
+    "## $ hwhm\\_collisions=\\gamma_{self} \\cdot P  $"
```

Comment:
could be written Sigma instead of HHM
```
$ \\sigma\\_{collisions}=\\gamma_{self} \\cdot P $
```

Define sigma in the text before ? 
"where $\\sigma$ is the half-width at half maximum (HWHM) of the Lorentzian profile (in cm-1)"